### ISSUE TITLE
Create a vignette to demo the integration between SS3 and ASAR

### ISSUE THREAD

**Issue by Bai-Li-NOAA** [nmfs-ost/asar#46]
Create a demo to show how to convert output from SS3 using ASAR and generate a report. Include report snippets in the vignette for visualization would be great. We can reuse the test models from tests/testthat/fixtures and the code in tests/testthat/fixtures/prepare_ss3_output_files.R and tests/testthat/test-convert_output.R to streamline the integration.

**Comment by Schiano-NOAA** [comment]
@Bai-Li-NOAA not sure if this works OK, but I made a tutorial for asar that does this. This let's the user follow along and provides the correct output for them as well. Here are the sites:

[GitHub repo](https://github.com/nmfs-ost/asar-demo)
[Shiny App](https://connect.fisheries.noaa.gov/asar_tutorial/)

**Comment by Schiano-NOAA** [comment]
@Bai-Li-NOAA thanks for reviewing! I also noticed that the images weren't loading. Also adding the clipboard button is a great suggestion! Will definitely be adding that

**Comment by sbreitbart-NOAA** [comment]
Was this resolved with the creation of the tutorial?

**Comment by Schiano-NOAA** [comment]
@sbreitbart-NOAA yes! I fixed the images issue. I couldn't figure out how to add the copy buttons though. I think that feature isn't available through the tutorial template I have/I don't know how to add it in lol
